cupy/_manipulation/basic.py

```diff
@@ -35,6 +36,14 @@ def copyto(dst, src, casting='same_kind', where=None):
     if src_is_python_scalar:
         src_dtype = numpy.dtype(type(src))
         can_cast = numpy.can_cast(src, dst.dtype, casting)
+    elif isinstance(src, numpy.ndarray):
+        if src.shape != ():
```

Comment:
ndarray.item works when src.size == 1?

Reply:
Yes, passing no arguments to ndrray.item is for that purpose. (Or you say another thing?)

https://numpy.org/doc/stable/reference/generated/numpy.ndarray.item.html
> none: in this case, the method only works for arrays with one element (a.size == 1), which element is copied into a standard Python scalar object and returned.

Reply:
How about changing this condition to `src.size == 1`? We can support also `(1,)` shape arrays.

Reply:
Sure!